.github/ISSUE_TEMPLATE/create_release_branch.md

```diff
@@ -89,7 +89,7 @@ The steps are to be followed in-order, each task must be completed by the person
 - [ ] **Owner**: Create `release-1.xx` branch from latest `main` in rawfile-localpv
   - `git clone git@github.com:canonical/rawfile-localpv.git ~/tmp/release-1.xx`
   - `pushd ~/tmp/release-1.xx`
-  - `git switch main`
+  - `git switch rockcraft`
```

Comment:
huh? Why do we have this weird main branch?

Reply:
this is the "default" branch of the rawfile-localpv b/c this is a forked repo and rather than working out of main (as the upstream author does) we do our work out of the rockcraft branch.  We're trying to get the rockcraft work upstreamed...

so for the moment "OUR" main is `rockcraft`